src/bicep/examples/automationAccount/automationAccount.bicep

```diff
@@ -0,0 +1,64 @@
+/*
+Deployes an automation account with modules/runbook/diagnostic logs
+Makes use of example azure automation bicep: https://github.com/Azure/bicep/tree/main/docs/examples/301/automation-account-import-runbooks-and-modules
+*/
+targetScope = 'subscription'
+
+param mlzDeploymentVariables object = json(loadTextContent('../deploymentVariables.json'))
+
+@description('The name of the automation account which will be created.  If unchanged or not specified, the MLZ resource prefix + "-aAc" will be utilized.')
+param automationAcctName string = '${mlzDeploymentVariables.mlzResourcePrefix.Value}-aAc'
+
+@description('The name of the resource group in which the automation account will be deployed. If unchanged or not specified, the MLZ operations resource group is used.')
+param targetResourceGroup string = '${mlzDeploymentVariables.spokes.Value[1].resourceGroupName}'
+
+@description('The URL location to the powershell runbook you wish to use')
+param automationRunbook string = 'https://raw.githubusercontent.com/Azure/missionlz/breestryker/AzAutomationBicepEx/src/bicep/examples/automationAccount/test.ps1'
+
+@description('A string dictionary of tags to add to deployed resources. See https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources?tabs=json#arm-templates for valid settings.')
+param tags object = {}
+var defaultTags = {
+  'DeploymentType': 'MissionLandingZoneARM'
+}
+var calculatedTags = union(tags, defaultTags)
+
+var targetSubscriptionId_Var = targetResourceGroup == '${mlzDeploymentVariables.spokes.Value[1].resourceGroupName}' ? '${mlzDeploymentVariables.spokes.Value[1].subscriptionId}' : subscription().subscriptionId
+var location = deployment().location
+
+resource targetAAResourceGroup 'Microsoft.Resources/resourceGroups@2020-10-01' = {
+  name: targetResourceGroup
+  location: location
+}
```

Comment:
If the resource group already exists, this has the side effect of overwriting settings on the existing resource group. For example, I had tags applied to this resource group and this deployment removed them. Is there a way to use the existing RG if it's there or create a new one if it's not there?

Reply:
I think creating a new one is maybe out of scope for the example?  I will add an existing tag though.  A lot of this formatting I pulled out of other examples. We might need to update them as well.

Reply:
Switched to using existing resource.